mypy/stubgenc.py

```diff
@@ -237,13 +254,24 @@ def generate_c_function_stub(
 
             if is_overloaded:
                 output.append("@overload")
-            output.append(
-                "def {function}({args}) -> {ret}: ...".format(
-                    function=name,
-                    args=", ".join(sig),
-                    ret=strip_or_import(signature.ret_type, module, imports),
+            if include_docstrings and docstr:
+                output.append(
+                    "def {function}({args}) -> {ret}:".format(
+                        function=name,
+                        args=", ".join(sig),
+                        ret=strip_or_import(signature.ret_type, module, imports),
+                    )
+                )
+                docstr_indented = "\n    ".join(docstr.strip().split("\n"))
+                output.extend(f'    """{docstr_indented}"""'.split("\n"))
```

Comment:
nit: can't necessarily assume that `"""` is a valid quote. E.g., this will break if someone's docstring is `''' ..."""... '''` or `' ends with quote"'`

Reply:
I think we also need to be very careful with testing things like `""" \" """` and other `\` chars in docstrings. Maybe even falling back to `r"""` if needed.

Reply:
Escaping also wasn't handled correctly. I've updated the code to use `repr` of the docstring. That should solve all these issues.